Discord Update

chris-mercer opened the pull request:
Update to a discord server that we can maintain with auto moderation, anti-spam, scammer protections and such. Current server is in a deprecated state and has been for a couple of years. Now to the detriment to the project. Retire it as a legacy server and build a first class chat experience for ETC.

chris-mercer committed:
Discord Update

Update to a discord server that we can maintain with auto moderation, anti-spam, scammer protections and such. Current server is in a deprecated state and has been for a couple of years. Now to the detriment to the project. Retire it as a legacy server and build a first class chat experience for ETC.

chris-mercer committed:
More Index Words

Add more words to the word bank

IstoraMandiri commented:
Please lets keep PRs specific to one thing at a time. This PR not only updates the discord link and redirect, but also updates the landing page words, which will also have side-effects in second languages. The word change should be a separate PR.

Since you are an Admin of the proposed new server, I would also like to hear some unbiased input from other community members about this before approving.

I was only using the legacy server, but am open to changing if others are okay with it. 

Thank you

IstoraMandiri commented:
Another option is for us to have a new page for /discord, so instead of redirection, the user is given a choice of invite links.

chris-mercer committed:
Revert "More Index Words"

This reverts commit 037379ce54365d1804065defb1b4c6b5205d4aa3.

IstoraMandiri commented:
I agree with the logic and appreciate the efforts, so unless there's are any objections I can't see why not.

I've asked for feedback in the old server and so far no objections.

For transparency, and to avoid confusion, do you think you could craft a brief blog post explaining the the change and add it to this PR? 

It would be good if you could mention the new admins and moderation policy. I bring this up because I hope that you can commit to open debate even if it's against the opinion of the Admins.

A blog post might also encourage people to Join the new server. 

As far as I can see the main issue with the legacy server is that the admin became difficult to reach, which made it difficult to implement modernizations like on-boarding. Having an active Admin allows new things to be done and features to be implemented.

I still think it's good to have the legacy server around as a fallback.

chris-mercer committed:
Add Blog Post

Add blog about discord update.

realcodywburns commented:
I agree with Istora. Overall I am for a channel with active moderation other than hero efforts. I think social consensus building and coordination need a place welcoming to new users and veterans of the old chain wars. 


Transparent governance is must. 
Active moderation and engagement. 
Not a bot world


We can discuss this further on the call tomorrow.

IstoraMandiri committed:
Clarify recommendations for Discord server migration

Updated language to clarify the status and recommendations regarding the new Ethereum Classic Community Discord server. Adjusted phrasing to emphasize the encouragement for community members to migrate to the new server while maintaining access to the legacy server.

realcodywburns reviewed (approved):
LGTM

IstoraMandiri commented on content/ui.global.yaml:
```diff
@@ -83,11 +83,11 @@ socialItems:
-    __link: https://github.com/etclabscore
+    name: ETC Network Github
+  githubEtcClient:
+    __link: https://github.com/ethereumclassic/core-geth
```
For the record, this is what slipped passed review. 

Much more controversial and definitely not a "discord update"

chris-mercer replied:
What is controversial about the client link pointing to the project's repo again?

IstoraMandiri replied:
In the context of this this PR titled "Discord Update", it is much more controversial than simply updating discord, since there are significant downstream consequences for node operators trying to determine which version of the software to run, and the implications of who has write access to those repos. 

For the discussion about changing the core geth repo:

https://github.com/ethereumclassic/ethereumclassic.github.io/pull/1678

chris-mercer replied:
Again, what is controversial about the client link pointing to the project's repo?

From: https://github.com/ethereumclassic/ethereumclassic.github.io/pull/1678
> This PR updates the website to point to ethereumclassic/core-geth as the canonical client repository, replacing the current link to etclabscore/core-geth and remove the link to the etclabscore repo from the footer.
> 
> Rationale
> The ETC core geth client is a community asset. Many other core ETC repository — ECIPs, the website, community calls — lives in the ethereumclassic org where changes require review from multiple maintainers before merging. The client is the exception, and that exception creates a governance gap.
> 
> The recent https://github.com/etclabscore/core-geth/pull/694 security release illustrates the gap concretely. It was a valid and necessary fix, but it was merged to the production client without prior visibility to other contributors or community maintainers. For a security patch that's an understandable tradeoff. As a general operating model for the canonical client, it isn't.
> Moving the canonical pointer to the community org establishes that:
> 
> Official releases are visible to core contributors before they reach the website and CI/CD pipelines
> The client has the same oversight model as every other ETC community repository
> No single institution's infrastructure is a single point of failure for the community's canonical client reference
> Scope
> This PR changes the website link. CI/CD pipeline migration to watch the ethereumclassic org repo is a follow-on coordination item and does not block this change. The intent is to establish the governance standard; the technical migration is low-lift and can follow.
> 
> This is not a personnel change. The Coop's role maintaining the client and the bootnodes is unchanged. This is about where the community's official reference to that work lives.
